doc/source/data/dataset-tensor-support.rst

```diff
@@ -58,9 +59,11 @@ This section shows how to create single and multi-column Tensor datasets.
 
 .. tabbed:: Parquet
 
-  There are two ways to construct a parquet Tensor dataset: (1) loading a previously-saved Tensor
-  dataset, or (2) casting non-Tensor parquet columns to Tensor type. When casting data, a tensor
-  schema or deserialization UDF must be provided. The following are examples for each method.
+  There are two ways to construct a parquet Tensor dataset: (1) loading a
+  previously-saved Tensor dataset, or (2) casting non-Tensor parquet columns to Tensor
+  type. When casting data, a tensor schema or deserialization
+  :ref:`user-defined function <transform_datasets_writing_udfs>`  must be provided. The
```

Comment:
This might be confusing since this is referring to writing a read-time block UDF, while the cross-referenced doc section is talking about writing UDFs for the `.map()` and `.map_batches()` APIs, so the pointer isn't quite appropriate.

Reply:
Not at the moment, no, since UDFs are primarily a `.map()` and `.map_batches()` thing. This is an advanced concept and we're planning on deprecating `_block_udf`, so I think we could leave this as is.